frontend/src/Editor/Components/Divider/icons/HorizontalDivider.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+
+export default ({ fill, width}) => {
+  return (
+    <svg version="1.1" id="Capa_1"  x="0px" y="0px"
```

Comment:
Instead of an SVG, we should use the divider from tabler: https://preview.tabler.io/docs/divider.html
Also, I don't think a vertical divider is needed at the moment.